### PR TITLE
feat : Added a confirmation dialog while disabling the account

### LINKED ISSUE #460: Add a confirmation dialog while disabling the account
**Summary:**

Disable account is an operation which should have a confirmation dialog. Create a confirmation dialog when clicked on disable account button.

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #460 

Please Add Screenshots If any UI changes.

Screenshot_2019-03-10-20-26-44-119_org mifos mobilewallet mifospay

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@naman14 @luckyman20 Please review this PR.

**Comment by RajaVamsi11** [comment]
@droidchef Can you please review this PR.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/ui/SettingsActivity.java [hunk 1]
```diff
@@ -45,6 +47,24 @@ public void onLogoutClicked() {
 
     @OnClick(R.id.btn_disable_account)
     public void onDisableAccountClicked() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(this.getString(R.string.disable_this_account));
+        builder.setCancelable(false);
+        builder.setPositiveButton(R.string.confirm, new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+
```
Comment (shiv07tiwari) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/ui/SettingsActivity.java]: Add disable account here when user clicks on YES.
Also, resolve merge conflicts. @RajaVamsi11

**Comment by luckyman20** [comment]
Closing this because of inactivity.

**Pull request closed without merging**